lib/failure.sh

```diff
@@ -607,6 +607,17 @@ warn() {
   echo ""
 }
 
+warn_aws_proxy() {
+  local build_dir=${1:-}
+  local HTTP_PROXY HTTPS_PROXY NO_PROXY
+  HTTP_PROXY="$build_dir/HTTP_PROXY"
+  HTTPS_PROXY="$build_dir/HTTPS_PROXY"
+  NO_PROXY="$build_dir/NO_PROXY"
```

Comment:
Do we need to set/reference these, or can we reference them as they are?

Reply:
shellcheck gave a [literal string error](https://github.com/koalaman/shellcheck/wiki/SC2157) and referencing it like this was the only thing that worked. Do you know of any other way to do it?

Reply:
I see, if you delete lines 611 - 615, you can access them as global variables and they will pass shellcheck.